### DISCUSSION TITLE
[Custom]: How to configure zsh tty with wallust

### DISCUSSION THREAD

**Discussion by NischalDawadi** [JaKooLit/Hyprland-Dots#272] [Q&A] [answered]
Zsh tty theme is configured for pywal and doesn't take the colorscheme for wallust.

**Comment by JaKooLit** [comment]
woops... forgot to remove that one..

thanks bro

**Comment by JaKooLit** [comment]
https://github.com/JaKooLit/Fedora-Hyprland/commit/93de942fd9fddf5ce261578bed56e929f703fce7
https://github.com/JaKooLit/Arch-Hyprland/commit/84e32d5ea376568906b712b1ead6bc4b073abb95
https://github.com/JaKooLit/Debian-Hyprland/commit/37c48fbacfbac2e1f188d0f2a687a86f2371509c
https://github.com/JaKooLit/Debian-Hyprland/commit/d3d203c949e805885df7b87613054efdf6d01416
https://github.com/JaKooLit/OpenSuse-Hyprland/commit/5a62d9dd9394630909ce6ab4496b72d19830b10f

Im gonna edit my wiki to remove pywal and if want to use for kitty and tty...

**Comment by NischalDawadi** [comment]
wallust-sww doesnot change terminal colors any way to enable it.
i have to manually run wallust to change terminal color everytime i change the wallpaper.

**Comment by JaKooLit** [comment, marked as answer]
youre on the right file

just remove 
the -s on line 36
wallust run "$wallpaper_path" -s &

**Comment by JaKooLit** [comment]
then you can edit all the wallpaper scripts inside UserScripts

remove all the -s and just wallust run

**Comment by JaKooLit** [comment]
I will update the wiki

**Reply by NischalDawadi** [reply]
I got it work playing randomly and just saw your message.
Thank you. 
But i noticed wallust creates different color-schemes than pywal from same wallpaper.

**Comment by JaKooLit** [comment]
you can play around the settings.. I do believe reading wallust can use pywal as backend, although discourage

https://codeberg.org/explosion-mental/wallust/wiki/wallust.1-Man-Page

https://codeberg.org/explosion-mental/wallust/src/branch/dev

**Comment by JaKooLit** [comment]
wallust toml is located at

~/.config/wallust/wallust.toml

we can actually make template on wallust

at some point, im gonna integrate something to have a universal color scheme

**Comment by NischalDawadi** [comment]
Its quite difficult migration.
I was using pywalfox for firefox and wal for vscode as well as telegram
Any alternatives for them?

**Reply by JaKooLit** [reply]
need to start to stay away from pywal since its already been archived....

**Comment by JaKooLit** [comment]
https://github.com/JaKooLit/Hyprland-Dots/wiki/FAQ#%EF%B8%8F-wallust-color-changed-on-tty-or-kitty
